docs/source/ko/_toctree.yml

```diff
@@ -11,12 +11,13 @@
     title: (번역 중)
   title: 튜토리얼
 - sections:
-  - local: in_translation
-    title: (번역 중)
+    - local: sagemaker
+      title: Amazon SageMaker에서 학습 실행하기
+    - local: in_translation
+      title: (번역 중)
+  title: 일반 사용법
   title: How-to 가이드
```

Comment:
We should modify these lines to:
```yml
- sections:
  - sections:
    - local: sagemaker
      title: Amazon SageMaker에서 학습 실행하기
    - local: in_translation
      title: (번역 중)
    title: 일반 사용법
  - local: in_translation
    title: (번역 중)
  title: How-to 가이드
```
I understand the previous comment about how removing a single dash would resolve the issue might be misleading. Sorry for any confusion I may have caused.
Please let me know if you have further questions, @jungnerd !